src/databaseConnection.ts

```diff
@@ -1,29 +1,29 @@
 import pg from 'pg'
 import { user, password } from './secrets'
 // pq is a CommonJS module, so we have to do it this way for the import to work
```

Comment:
Did you mean 'pg' instead of 'pq'?

Reply:
Very possibly, though this isn't my code xD will fix it